Argo/Operators/DecodeDecoded.swift

```diff
@@ -1,43 +1,150 @@
-// MARK: Values
+/**
+  Attempt to decode a value at the specified key into the requested type.
 
-// Pull value from JSON
+  This operator is used to decode a mandatory value from the JSON. If the
+  decoding fails for any reason, this will result in a `Failure` being
+  returned.
+
+  - parameter json: The `JSON` object containing the key
+  - parameter key: The key for the object to decode
+
+  - returns: A `Decoded` value representing the success or failure of the
+             decode operation
+*/
 public func <| <A where A: Decodable, A == A.DecodedType>(json: JSON, key: String) -> Decoded<A> {
   return json <| [key]
 }
 
-// Pull optional value from JSON
+/**
+  Attempt to decode an optional value at the specified key into the requested
+  type.
+
+  This operator is used to decode an optional value from the JSON. If the key
+  isn't present in the JSON, this will still return `Success`. However, if the
+  key exists but the object assigned to that key is unable to be decoded into
+  the requested type, this will return `Failure`.
+
+  - parameter json: The `JSON` object containing the key
+  - parameter key: The key for the object to decode
+
+  - returns: A `Decoded` optional value representing the success or failure of the
+             decode operation
+*/
 public func <|? <A where A: Decodable, A == A.DecodedType>(json: JSON, key: String) -> Decoded<A?> {
   return .optional(json <| [key])
 }
 
-// Pull embedded value from JSON
+/**
+  Attempt to decode a value at the specified key path into the requested type.
+
+  This operator is used to decode a mandatory value from the JSON. If the
+  decoding fails for any reason, this will result in a `Failure` being
+  returned.
+
+  - parameter json: The `JSON` object containing the key
+  - parameter keys: The key path for the object to decode, represented by an
```

Comment:
Perhaps add an example to show what a key path should look like? I don't think immediately intuitive, especially considering the normal meaning of a key path in the APIs is a dot separated string.